app/src/main/java/ch/sdp/vibester/activity/LyricsBelongGameActivity.kt

```diff
@@ -20,6 +21,7 @@ private const val REQUEST_AUDIO = 100
 
 class LyricsBelongGameActivity : AppCompatActivity() {
     private lateinit var speechInput : String
+    private val baseUrl = "https://api.lyrics.ovh/"
```

Comment:
same here: Shouldn't be BASEURL ?

Reply:
i have missed the comment. I will add it in my other pr.